src/System.IO.Abstractions.TestingHelpers/MockDirectoryInfo.cs

```diff
@@ -131,7 +131,15 @@ public override DateTime LastWriteTimeUtc
             set { GetMockFileDataForWrite().LastWriteTime = value.ToLocalTime(); }
         }
 
+#if NET6_0_OR_GREATER
```

Comment:
We currently use dedicated "per-feature" constants for such things (see https://github.com/TestableIO/System.IO.Abstractions/blob/main/Directory.Build.props#L13-L15). Would you please follow the same pattern (just for consistency)?

Reply:
Updated